cloud/google/machineactuator.go

```diff
@@ -103,6 +112,15 @@ func NewMachineActuator(kubeadmToken string, machineClient client.MachineInterfa
 		}
 	}
 
+	// TODO: get rid of empty string check when we switch to the new bootstrapping method.
```

Comment:
For each of these TODOs, it would be great to create a tracking issue with all the necessary context, and link to them in these comments. Otherwise, the actual meaning of "new bootstrapping method," and why an empty string check can be removed, etc. will slowly fade from people's memory.